NEWS.md

```diff
@@ -1,5 +1,35 @@
 MXNet Change Log
 ================
+## 0.11.0
```

Comment:
I don't think we can call it 0.11.0 before the vote succeeds. It should be 0.11.0-rc0